src/main/java/Duke.java

```diff
@@ -1,10 +1,13 @@
+import java.util.ArrayList;
 public class Duke {
```

Comment:
Java coding standard: Logical units should be separated by one blank line. 

Can consider leaving a line between the import statements and class definition in this case.